docs/03-github/04-builder.mdx

```diff
@@ -554,11 +554,13 @@ jobs:
           - Android # Build an Android .apk standalone app.
           - WebGL # WebGL.
     steps:
-      - uses: actions/checkout@v2
+      - if: matrix.targetPlatform == 'Android'
+        uses: jlumbroso/free-disk-space@v1.2.0
```

Comment:
Personally I'd put this before the job that actually requires it. Also so that possible other steps (like in the other comment) can run unencumbered.

I believe it will both be easier to understand the workflow like that (i.e. people might not skip that part in copy-pasting) and keep the workflows as fast as possible.

Reply:
Moved too